griptape/tasks/prompt_task.py

```diff
@@ -24,10 +25,36 @@ class PromptTask(BaseTextInputTask):
     )
 
     output: Optional[Union[TextArtifact, ErrorArtifact, InfoArtifact]] = field(default=None, init=False)
+    
+    @rulesets.validator
+    def validate_rulesets(self, _, rulesets: list[Ruleset]) -> None:
+        if not rulesets:
+            return
+
+        if self.rules:
+            raise ValueError("can't have both rulesets and rules specified")
+
+    @rules.validator
+    def validate_rules(self, _, rules: list[Rule]) -> None:
+        if not rules:
+            return
+
+        if self.rulesets:
+            raise ValueError("can't have both rules and rulesets specified")
 
     @property
     def all_rulesets(self) -> list[Ruleset]:
-        return self.structure.rulesets + self.rulesets
+        if self.structure.rulesets:
+            structure_rulesets = self.structure.rulesets
+        else:
+            structure_rulesets = [Ruleset(name="Structure Ruleset", rules=self.structure.rules)]
+
+        if self.rulesets:
+            task_rulesets = self.rulesets
+        else:
+            task_rulesets = [Ruleset(name="Task Ruleset", rules=self.rules)]
```

Comment:
We should check here if rules are not empty.

Reply:
...and let's rename this to `Additional Ruleset`?

Still doesn't quite feel right. Any other ideas?